checks/compound/erc20/mainnetERC20InfoLookup.json

```diff
@@ -1 +1,44 @@
-{"0x221657776846890989a759ba2973e427dff5c9bb":{"symbol":"REPv2","decimals":18},"0xdac17f958d2ee523a2206206994597c13d831ec7":{"symbol":"USDT","decimals":{"type":"BigNumber","hex":"0x06"}},"0x6b175474e89094c44da98b954eedeac495271d0f":{"symbol":"DAI","decimals":18},"0x0d8775f648430679a709e98d2b0cb6250d2887ef":{"symbol":"BAT","decimals":{"type":"BigNumber","hex":"0x12"}},"0x956f47f50a910163d8bf957cf5846d573e7f87ca":{"symbol":"FEI","decimals":18},"0x1f9840a85d5af5bf1d1762f925bdaddc4201f984":{"symbol":"UNI","decimals":18},"0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48":{"symbol":"USDC","decimals":6},"0xc00e94cb662c3520282e6f5717214004a7f26888":{"symbol":"COMP","decimals":18}}
\ No newline at end of file
+{
+  "0x221657776846890989a759ba2973e427dff5c9bb": {
+    "symbol": "REPv2",
+    "decimals": 18
+  },
+  "0xdac17f958d2ee523a2206206994597c13d831ec7": {
+    "symbol": "USDT",
+    "decimals": {
+      "type": "BigNumber",
+      "hex": "0x06"
```

Comment:
Why do we save hex here? This will cause issues on retrieval

Reply:
We should convert it to decimal format and then save it

Reply:
Resolved